vaadin-testbench-core/src/main/java/com/vaadin/testbench/unit/ComponentQuery.java

```diff
@@ -114,6 +118,56 @@ public ComponentQuery<T> withCondition(Predicate<T> condition) {
         return this;
     }
 
+    /**
+     * Requires the components to have all the given CSS class names
+     *
+     * @param className
+     *            required CSS class name, not {@literal null}
+     * @param other
+     *            additional required CSS class names
+     *
+     * @return this element query instance for chaining
+     */
+    public ComponentQuery<T> withClassName(String className, String... other) {
```

Comment:
Should there be a note that the order did matter or is it irrelevant? Looking below with the Hashset and String.join that looks like a culprit for unexpected behavior

Reply:
The order here doesn't matter, since under the hood every single class name is checked  separately

Reply:
Thanks! With that in mind - would it make sense to add a test that 

withClassName("test-class other-class") would work as well as little convenient for the developer? There were similar additions to flow in the past to easy development with multiple css classes